Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -28,6 +28,16 @@ void LayoutAnimationsManager::configureAnimation(
   }
 }
 
+void LayoutAnimationsManager::disableExiting(int tag) {
+  auto lock = std::unique_lock<std::mutex>(animationsMutex_);
```

Comment:
Please dig if we actually need this mutex here and why (testing it without it and breaking it is sufficient 😅).